_posts/2020-04-08-secure-software-development-lifecycle.md

```diff
@@ -0,0 +1,138 @@
+* Fuzz testing
+* Static code analysis for your source code
+* Best practice analysis in some cases
+
+You should not forget the manual testing. For example:
+* Review what others have done
+* Exploratory testing
+* Penetration testing
+* Trying out disaster recovery practices
+
+#### Deployment
+
+The moment you publish your source code as a software it becomes legacy and you need to support it. From SSDLC perspective this means that you need to actively do the things that you need to have processes for. It also means that you have published your software open for use. You will get feedback and you need to have processes to deal with it. You can gather insights of the usage by monitoring the software but you could also make your software open for hacking with bug bounty program. Minimum requirement is that you know how to process feedback.
+
+* Change management
+* Reading news (and vulnerability informations)
+* Incident and response
+* Monitoring and alerting
+* Bug bounty
+
+#### Retirement
+
+This is often forgotten in all projects. Some day the time of our beloved system will come to an end. At this point the system has had a great deal of organical growth and understanding the implications of the retirement can be hard to see. System needs to be taken down in a managed manner so that business can move to a new world, but at the same time all the resources of the system need to be disposed in a governed manner. Failing to due so will lead into an information leak. 
+
+Things that you should consider:
+* Migration plan
+* Archiving data (there might be laws)
+* Destroying virtual or physical resources
+* Cleaning up the data stores
+* Cleaning all the DNS names and such to avoid subdomain takeovers (especially with cloud)
+
+
+#### Example implementation
+
+People at Unity Tech have been so awesomely kind that they have published their SSDLC as [open source](https://github.com/UnityTech/unity-ssdlc/blob/master/Overview.md). I would strongly recommend looking at it as an example of an implementation.
+
+## My experience on the matter
+
+Taking security oriented tasks on the backlog has been natural since security has been brought up in front. Communication between different parties has been easy since we have had structure and documentation for the security conversations to take place. For example not a single threat modeling session has been in vain. It might feel before that we already know everything but there is always something we have missed.
+
+Key factor for making secure software development lifecycle to work is to find a spokesperson within your project for the matter. It does not need to be an extra guy but just somebody who likes to be a spokesperson for security matters. This person does not need know everything about information security as software development is always team work. It is just a good idea to have somebody who keeps on track about all security related solutions in your project and understands the greater picture.
```

Comment:
an extra guy <- "a person outside the project", perhaps?

Reply:
did not like that version. I meant that you do not need to introduce new guys to the project as the role can be virtual. I guess I stick with my version.